frontend/src/views/Pages/RegisterCommunity.spec.js

```diff
@@ -1,6 +1,8 @@
 import { mount, RouterLinkStub } from '@vue/test-utils'
 import RegisterCommunity from './RegisterCommunity'
 
+import { toasters } from '../../mixins/toaster'
```

Comment:
cant we do this globally?
Including the spy function?

This would be the same behaviour as the toasts in the application which are also defined globally

Reply:
Good idea!